iothub/device/src/ClientOptions.cs

```diff
@@ -13,5 +13,13 @@ public class ClientOptions
         /// This feature is currently supported only over MQTT.
         /// </summary>
         public string ModelId { get; set; }
+
+        /// <summary>
+        /// The transport settings to use for all file upload operations, regardless of what protocol the device
+        /// client is configured with. All file upload operations take place over https. If no client certificates
+        /// are configured to this object, then file upload operations will use the client certificates configured
+        /// in the transport settings set for the non-file upload operations.
+        /// </summary>
+        public Http1TransportSettings FileUploadTransportSettings { get; set; } = new Http1TransportSettings();
```

Comment:
Can I use these settings if I need to use HTTP on device client?

Reply:
No. They are strictly for file upload operations, hence the name

Reply:
You can provide the same instance of HTTP transport settings for the base device client operations and for this file upload transport settings option, though

Reply:
That would confuse the customer. If I end up providing Http1TransportSettings  then I would ideally want it to work on messaging as well if I use HTTP.

Reply:
That hasn't been the behavior in the past, so what you are suggesting would be a breaking change. It isn't an option